Lesson_03/solution_task_n2.py

```diff
@@ -0,0 +1,10 @@
+# 2. Create a program that will print “I love Python” to the console every 4.2 seconds until its execution is manually
+# interrupted.
+
+import time
+
+
+while True:
+
+    time_out = time.sleep(4.2)
```

Comment:
You can use it simply `time.sleep(4.2)`. There is no need to save the result in `time_out` variable